manifests/00-custom-resource-definition.yaml

```diff
@@ -766,12 +416,9 @@ spec:
                 controller pods corresponding to the IngressController. The number
                 of matching pods should equal the value of availableReplicas.
               type: string
-          required:
-          - availableReplicas
-          - selector
-          - domain
```

Comment:
@Miciah hmmm

Reply:
This is in `status`, so I don't know that we care...?

Reply:
I don't know how to think about this one. Maybe @deads2k can help or knows who can.